samcli/lib/providers/provider.py

```diff
@@ -481,7 +481,7 @@ def binary_media_types(self) -> List[str]:
 
 class Cors(_CorsTuple):
     @staticmethod
-    def cors_to_headers(cors: Optional["Cors"]) -> Dict[str, Union[int, str]]:
+    def cors_to_headers(cors: Optional["Cors"], request_origin: str) -> Dict[str, Union[int, str]]:
```

Comment:
nit: Can we have the optional parameter come after the required parameters?

Reply:
`request_origin` is actually optional as well - will fix